src/material-experimental/mdc-tooltip/tooltip.scss

```diff
@@ -11,4 +14,8 @@
   // depending on the state of the overlay. For tooltips the overlay panel should never enable
   // pointer events. To overwrite the inline CSS from the overlay reference `!important` is needed.
   pointer-events: none !important;
+
+  @include tooltip-theme.fill-color(
+    map.get(palette.$light-theme-background-palette, tooltip)
```

Comment:
We should have a comment about why it's done this way. Also shouldn't these styles be in the tooltip theme?

Reply:
I thought we kept all of the custom styles inside the `<component>.scss`, no?

Reply:
No, we can have custom styles anywhere, but the properties based on the theme (e.g. colors) need to be in the theme file.